crates/postgres-docker-utils/src/lib.rs

```diff
@@ -92,7 +92,7 @@ fn parse_exposed_port(s: &str) -> anyhow::Result<SocketAddr> {
         .iter()
         .map(|p| p.parse::<SocketAddr>())
         .next()
-        .context("Missing exposed port")??)
+        .context("Missing exposed port, is your docker daemon running?")??)
```

Comment:
Nice!

Reply:
🙃

Reply:
you wouldn't believe how many times I tried running signup sequencer tests and didn't have docker running😂 thanks for this